src/07-even-more-loops.js

```diff
@@ -96,7 +134,9 @@ function evenArray(array) {}
  *  findSmallest([ 0, 11, -2, 5 ]);
  *  //> -2
  */
-function findSmallest(array) {}
+function findSmallest(array) {
+  return array.sort()[0];
```

Comment:
`.sort()` has some interesting implementations.  Could you alter this function to sort our array the other direction?